### PR TITLE
We don't need 5 separate variables to know which station we are on

### PR DESCRIPTION
**Pull request by RyanSmake**
Eliminates some of them (see the title). I wasn't sure how to remove all.

Also, everything normally should use station_name() now to get the station's name. Everything I did not delete does not affect gamepley directly.

Bonus:
Central Command Reports now respect CC's name. In the case of the Cyberiad we now get NAS Trurl Reports (actual announcements are unnaffected, printed papers however will now carry the name).

### CODE REVIEW

File: code/_globalvars/configuration.dm [hunk 1]
```diff
@@ -2,7 +2,6 @@ var/datum/configuration/config = null
 
 var/host = null
 var/join_motd = null
-var/station_name = "NSS Cyberiad"
```
Comment (Tayyyyyyy) [inline code/_globalvars/configuration.dm]: Isn't this the string that comes after "Paradise Station -- " on the byond hub? Did you make sure that this doesn't break the byond hub entry?

Comment (SamHPurp) [reply code/_globalvars/configuration.dm]: You are right, and really this should trigger the station name, rather than be hard coded.

Comment (RyanSmake) [reply code/_globalvars/configuration.dm]: I did break it, thanks for the heads up. Should be fixed now.

File: code/__HELPERS/names.dm [hunk 2]
```diff
@@ -44,69 +31,18 @@ var/religion_name = null
...
 
 /proc/station_name()
-	if(station_name)
-		return station_name
-
-	var/name = ""
-
 	if(config && config.server_name)
-		world.name = "[config.server_name]: [name]"
+		world.name = "[config.server_name]: station_name()"
```
Comment (tigercat2000) [inline code/__HELPERS/names.dm]: I feel like a recursive call is perhaps a bad idea here

Comment (RyanSmake) [reply code/__HELPERS/names.dm]: The syntax is incorrect, there is no recursion there. Or any proc call for that matter. *facepalm*

File: code/modules/shuttle/supply.dm [hunk 3]
```diff
@@ -330,7 +330,7 @@
 	slip.points = object.cost
 	slip.ordernumber = ordernum
 
-	var/stationName = (errors & MANIFEST_ERROR_NAME) ? new_station_name() : station_name()
```
Comment (tigercat2000) [inline code/modules/shuttle/supply.dm]: This needs to stay, it factors into cargo's selling ability.

**Comment by tigercat2000** [comment]
There's tabbing issues in your last build.

**Pull request closed without merging**